src/clj/bluegenes_tool_store/package.clj

```diff
@@ -0,0 +1,33 @@
+(ns bluegenes-tool-store.package
+  (:require [bluegenes-tool-store.tools :as tools]
+            [cheshire.core :as cheshire]
+            [clj-http.client :as client]
+            [taoensso.timbre :refer [infof]]))
+
+(defn get-all-npm-tools
+  "Send an HTTP request to the NPMJS API to get a list of bluegenes tools."
```

Comment:
What happens if npm's api service isn't available? We could theoretically try falling back on another cdn if there is one (or does, say, yarn just use npm anyway?) but a simple error message is probaby sufficient.